examples/library/src/Server.hs

```diff
@@ -11,18 +11,19 @@
 module Main where
 
 import           Control.Monad.IO.Class            (liftIO)
-import           Control.Monad.Logger
-import           Data.Conduit
+import           Control.Monad.Logger              (LoggingT, runStderrLoggingT)
+import           Data.Conduit                      (ConduitT, Void, runConduit, (.|))
 import           Data.Maybe                        (fromJust)
 import qualified Data.Text                         as T
-import           Database.Persist
 import           Database.Persist.Sqlite
 import           Mu.Adapter.Persistent             (runDb)
-import           Mu.GraphQL.Quasi
-import           Mu.GraphQL.Server
-import           Mu.Instrumentation.Prometheus
-import           Mu.Schema
-import           Mu.Server
+import           Mu.GraphQL.Server                 (graphQLApp, liftServerConduit)
+import           Mu.Instrumentation.Prometheus     (initPrometheus, prometheus)
+import           Mu.Schema                         (Mapping ((:->)), Proxy (Proxy))
+import           Mu.Server                         (ServerError (ServerError),
```

Comment:
If any, I would not use explicit exports for the `Mu` things, the list is too long and we use almost everything.